src/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/ArtworkDetails.tsx

```diff
@@ -128,11 +129,11 @@ export const ArtworkDetails: React.FC<ArtworkDetailsProps> = ({
         utmSource: utmParams?.utmSource,
         utmTerm: utmParams?.utmTerm,
         sessionID: !isLoggedIn ? getENV("SESSION_ID") : undefined,
+        myCollectionArtworkID: artworkId && isFirstStep ? artworkId : undefined,
```

Comment:
**nitpick:** it would be great to add a comment here on why this is important. I would even go to add add a test to avoid this regression in the future

Reply:
Thanks. True, a test would make sense.